src/doc/reference.md

```diff
@@ -1983,6 +1983,7 @@ macro scope.
 
 ### Miscellaneous attributes
 
+- `deprecated` - Allows using the `#[deprecated]` attribute.
```

Comment:
Did you put this in the wrong place? I don't understand what this is saying or why it belongs in this list. It used to be that `#[feature(deprecated)]` allowed use of `#[deprecated]`, but it's stabilized now. Or did you mean the `#[allow(deprecated)]` attribute? But that allows usage of items marked deprecated, not use of `#[deprecated]` itself AFAIK.